docs/guides/spmd_api.ipynb

```diff
@@ -0,0 +1,1135 @@
+{
```

Comment:
<p>"scaled module output"  what does "scaled" mean here? do you mean "sharded" or am I missing something?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/flax/pull/2730/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
Removed "scaled" - hopefully it's clearer now.